src/glados/ws2es/es_util.py

```diff
@@ -438,15 +443,27 @@ def _recursive_simplify_es_properties(cur_dict: dict, cur_prefix: str):
             if 'es_mapping_leaf' in value.keys():
                 simple_props[next_prefix] = simplify_single_mapping(value)
             else:
+                simple_props[next_prefix] = {
+                    'type': 'object',
+                    'aggregatable': False,
```

Comment:
Include sortable property, it will be equivalent as aggregatable for elasticsearch properties, but it is useful for context properties like similarity score in molecule.